services/app-web/src/pages/Auth/Auth.test.tsx

```diff
@@ -84,46 +85,69 @@ describe('Auth', () => {
                         Pool: { getClientId: () => '7' } as CognitoUserPool,
                     })
                 )
-            const history = createMemoryHistory()
 
-            renderWithProviders(<CognitoLogin />, {
-                apolloProvider: {
-                    mocks: [
-                        fetchCurrentUserMock({ statusCode: 403 }),
-                        fetchCurrentUserMock({ statusCode: 403 }),
-                    ],
-                },
-                routerProvider: { routerProps: { history: history } },
-            })
+            let testLocation: Location
+
+            renderWithProviders(
+                <>
+                    <Route
+                        path="*"
+                        render={({ location }) => {
+                            testLocation = location as Location
+                            return null
+                        }}
+                    />
+                    (<CognitoLogin />
+                </>,
+                {
+                    apolloProvider: {
+                        mocks: [
+                            fetchCurrentUserMock({ statusCode: 403 }),
+                            fetchCurrentUserMock({ statusCode: 403 }),
+                        ],
+                    },
+                }
+            )
 
             await userLogin(screen)
 
             await waitFor(() => expect(loginSpy).toHaveBeenCalledTimes(1))
-            await waitFor(() => expect(history.location.pathname).toBe('/'))
+            await waitFor(() => expect(testLocation.pathname).toBe('/'))
         })
 
         it('when login fails, stay on page and display error alert', async () => {
             const loginSpy = jest.spyOn(CognitoAuthApi, 'signIn')
-            const history = createMemoryHistory()
-
-            renderWithProviders(<CognitoLogin />, {
-                apolloProvider: {
-                    mocks: [
-                        fetchCurrentUserMock({ statusCode: 403 }),
-                        fetchCurrentUserMock({ statusCode: 403 }),
-                        fetchCurrentUserMock({ statusCode: 403 }),
-                    ],
-                },
-                routerProvider: {
-                    route: '/auth',
-                    routerProps: { history: history },
-                },
-            })
+            let testLocation: Location
+
+            renderWithProviders(
```

Comment:
Every time I see this function it makes me happy. @haworku this testing function is brilliant, giving us usable defaults and making it easy to customize things per-test.